Sources/EmbraceConfigInternal/EmbraceConfigurable/RemoteConfig/RemoteConfigPayload.swift

```diff
@@ -148,6 +184,11 @@ public struct RemoteConfigPayload: Decodable, Equatable {
         networkSpansForwardingThreshold = 0.0
         uiLoadInstrumentationEnabled = true
 
+        metricKitEnabledThreshold = 0.0
+        metricKitCrashCaptureEnabled = false
+        metricKitCrashSignals = [CrashSignal.SIGKILL.stringValue]
```

Comment:
we'll have to wait and see, but iirc there are some MachO exceptions that don't have signals attached to them.